src/utils/cellSets/getCellClassProperties.js

```diff
@@ -1,33 +1,29 @@
-import _ from 'lodash';
-
 /**
- * Returns the properties of the first of the cellSets that contains the cellId
+ * Returns the properties of the cellSets that contains the cellId
  * (by the order specified in hierarchy children)
  * @param {int} cellId The id of the cell we are searching for.
- * @param {string} cellSetClassKey The cellSetClass we are searcing inside of
+ * @param {array} cellSetClassKeys The cellSetClass we are searching inside of
  *  (e.g.: sample, louvain, scratchpad, etc..).
- * @param {array} hierarchy CellSets hierarchy.
- * @param {object} properties CellSets properties.
+ * @param {object} cellSets  cellSets for the experiment.
  */
 
-const getCellClassProperties = (cellId, cellSetClassKey, hierarchy, properties) => {
-  const childrenCellSets = _.find(hierarchy, ({ key }) => key === cellSetClassKey).children;
-
-  let firstCellSetContainingCell = null;
-
-  childrenCellSets.every(({ key }) => {
-    if (properties[key].cellIds.has(cellId)) {
-      firstCellSetContainingCell = key;
+const getCellClassProperties = (cellId, cellSetClassKeys, cellSets) => {
```

Comment:
Sorry to make you add this change since it doesn't have to do much with your ticket, but do you think this name fits well with with what the function does?

Perhaps `getContainingCellSetsProperties` fits better instead?